packages/fiori/src/BarcodeScannerDialog.ts

```diff
@@ -10,6 +10,7 @@ import customElement from "@ui5/webcomponents-base/dist/decorators/customElement
 import property from "@ui5/webcomponents-base/dist/decorators/property.js";
 import event from "@ui5/webcomponents-base/dist/decorators/event.js";
 import type { Result, Exception } from "@zxing/library/esm5/index.js";
+import type ResultPoint from "@zxing/library/esm5/core/ResultPoint.js";
```

Comment:
Are you sure this type isn't exported by the "index" in the above line? Usually that's how libraries are structured.